str/str_iterative_eh_runner.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+"""
+Running Batch script to use ExpansionHunter to call STRs on 10 TOB-WGS genomes. 
```

Comment:
Update docstring, be good to include requirements here when testing, eg:

```
pip install sample-metadata hail click
```

And also an analysis-runner invocation about how to run. Eg: now that the egregions aren't hardcoded, where are they?